Gems/ROS2/Code/Source/VehicleDynamics/ModelLimits/SkidSteeringModelLimits.cpp

```diff
@@ -40,7 +42,21 @@ namespace ROS2::VehicleDynamics
                         "Angular speed Limit",
                         "Max angular speed (rad/s)")
                     ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
-                    ->Attribute(AZ::Edit::Attributes::Max, 10.0f);
+                    ->Attribute(AZ::Edit::Attributes::Max, 10.0f)
+                    ->DataElement(
+                        AZ::Edit::UIHandlers::Default,
+                        &SkidSteeringModelLimits::m_angularAccleration,
+                        "Angular acceleration",
+                        "Acceleration in rad/s²")
```

Comment:
Does the power of 2 symbol shows up correctly in the tooltip in the editor?

Reply:
Looks fine
![Screenshot from 2023-02-02 15-45-56](https://user-images.githubusercontent.com/3209244/216356349-eb274eac-f526-49b8-a068-90d6aef7c97e.png)

Reply:
Awesome! Thanks for verifying!